src/test/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/DeriveHearingCentreHandlerTest.java

```diff
@@ -97,22 +99,6 @@ void should_derive_hearing_centre_from_detention_facility_name_from_irc() {
         verify(bailCase, times(1)).write(HEARING_CENTRE, HearingCentre.HATTON_CROSS);
     }
 
-    @Test
```

Comment:
Instead of removing this test, you can update to test that new value should be updated on the field.

Reply:
Added it back in with changes